scheduler/core/components/selector/selector.py

```diff
@@ -55,8 +54,7 @@ class Selector(SchedulerComponent):
     collector: Collector
     num_nights_to_schedule: int
     time_buffer: TimeBuffer
-    cc_per_site: Dict[Site, CloudCover] = field(default_factory=lambda: {})
-    iq_per_site: Dict[Site, ImageQuality] = field(default_factory=lambda: {})
+    variant_per_site: Dict[Site, Variant] = field(default_factory=lambda: {})
```

Comment:
I already made this change in my last PR... it's `Selector._variant_snapshot_per_site: Dict[Site, VariantSnapshot]`. If you need to make it public, go ahead.